packages/wdio-sauce-service/src/service.js

```diff
@@ -90,6 +90,23 @@ export default class SauceService {
     }
 
     afterTest (test, context, results) {
+        /**
+         * remove failure if test was retried and passted
+         * > Mocha only
+         */
+        if (test._retriedTest && results.passed) {
+            --this.failures
+            return
+        }
+
+        /**
+         * don't bump failure number if test was retried and still failed
+         * > Mocha only
+         */
+        if (test._retriedTest && !results.passed && test._currentRetry < test._retries) {
```

Comment:
If we don't bump failures do we still need to remove it?

Reply:
nope, we only remove it if the test passed at the end